crash

```2019-02-09 20:51:28.245 [INF] BMGR: Processed 1 block in the last 6m3.16s (10 transactions, 4 tickets, 5 votes, 1 rev
ocation, height 317228, 2019-02-09 20:51:07 +0000 UTC)                                                               
2019-02-09 20:51:28.246 [INF] BMGR: Lost peer 113.252.46.114:56369 (inbound)                                         
panic: runtime error: invalid memory address or nil pointer dereference                                              
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x96cd2b]                                              
                                                                                                                     
goroutine 2419135 [running]:                                                                                         
main.handleGetPeerInfo(0xc010b39b30, 0xa00800, 0x11b9770, 0x0, 0xc000431b20, 0xc012103001, 0x4, 0xc0000a0750)        
        /home/dhill/.go/src/github.com/decred/dcrd/rpcserver.go:3358 +0x26b                                          
main.(*rpcServer).standardCmdResult(0xc010b39b30, 0xc001711e00, 0x0, 0xb, 0x11baa20, 0x462300, 0xc010ae5890)         
        /home/dhill/.go/src/github.com/decred/dcrd/rpcserver.go:5949 +0x98                                           
main.(*wsClient).serviceRequest(0xc0184ae400, 0xc001711e00)                                                          
        /home/dhill/.go/src/github.com/decred/dcrd/rpcwebsocket.go:1687 +0x28d                                       
main.(*wsClient).inHandler.func1(0xc0184ae400, 0xc001711e00)                                                         
        /home/dhill/.go/src/github.com/decred/dcrd/rpcwebsocket.go:1387 +0x35                                        
created by main.(*wsClient).inHandler                                                                                
        /home/dhill/.go/src/github.com/decred/dcrd/rpcwebsocket.go:1386 +0x2819  
```

--- a/server.go
+++ b/server.go
@@ -1704,6 +1704,7 @@ func (s *server) outboundPeerConnected(c *connmgr.ConnReq, conn net.Conn) {
 	if err != nil {
 		srvrLog.Debugf("Cannot create outbound peer %s: %v", c.Addr, err)
 		s.connManager.Disconnect(c.ID())
+		return
 	}
 	sp.Peer = p
 	sp.connReq = c

